### ISSUE TITLE
[Bug report]:  pymatgen-2023.11.10 can not work properly

### ISSUE THREAD

**Issue by Jeff-oakley** [materialsproject/pymatgen#3469]
### Problem

I recently update pymatgen using pip and then everything seems to be broken. The main reason is that there is a file missing under pymatgen.io.vasp

```
File "/Users/binouyang/miniconda3/envs/work3/lib/python3.9/site-packages/pymatgen/io/vasp/__init__.py", line 10, in <module>
    from .inputs import Incar, Kpoints, Poscar, Potcar, PotcarSingle, VaspInput
  File "/Users/binouyang/miniconda3/envs/work3/lib/python3.9/site-packages/pymatgen/io/vasp/inputs.py", line 1602, in <module>
    class PotcarSingle:
  File "/Users/binouyang/miniconda3/envs/work3/lib/python3.9/site-packages/pymatgen/io/vasp/inputs.py", line 1694, in PotcarSingle
    potcar_summary_stats = loadfn(POTCAR_STATS_PATH)
  File "/Users/binouyang/miniconda3/envs/work3/lib/python3.9/site-packages/monty/serialization.py", line 63, in loadfn
    with zopen(fn, "rt") as fp:
  File "/Users/binouyang/miniconda3/envs/work3/lib/python3.9/site-packages/monty/io.py", line 40, in zopen
    return bz2.open(filename, *args, **kwargs)
  File "/Users/binouyang/miniconda3/envs/work3/lib/python3.9/bz2.py", line 319, in open
    binary_file = BZ2File(filename, bz_mode, compresslevel=compresslevel)
  File "/Users/binouyang/miniconda3/envs/work3/lib/python3.9/bz2.py", line 85, in __init__
    self._fp = _builtin_open(filename, mode)
FileNotFoundError: [Errno 2] No such file or directory: '/Users/binouyang/miniconda3/envs/work3/lib/python3.9/site-packages/pymatgen/io/vasp/potcar_summary_stats.json.bz2
```

Do I need to install anything to make things work?

### Proposed Solution

N/A

**Comment by naik-aakash** [comment]
Seems a new release is needed. It has been fixed I guess on the git repo already.

**Comment by janosh** [comment]
Fixed in #3468 and released in [v2023.11.12](https://github.com/materialsproject/pymatgen/releases/tag/v2023.11.12).
